docs/configuration-reference/components/metallb.md

```diff
@@ -77,6 +77,46 @@ component "metallb" {
 
 MetalLB will use the specified CIDR for exposing services of type `LoadBalancer`.
 
+### Advanced IP allocation
+
+By default, MetalLB uses all specified address pools to allocate IP addresses to services. To
+request an address from a specific pool, set the `metallb.universe.tf/address-pool` annotation for
+the relevant service:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  annotations:
+    metallb.universe.tf/address-pool: pool2
+spec:
+  ports:
+  - port: 80
+    targetPort: 80
+  selector:
+    app: nginx
+  type: LoadBalancer
+```
+
+To exempt a pool from automatic IP allocation, set `auto-assign` on the pool to `false`:
+
+```yaml
+address-pools:
```

Comment:
Where is this located? In the MetalLB config map? In that case we should mention where this should go.

Reply:
ping @johananl

Reply:
> Where is this located? In the MetalLB config map? In that case we should mention where this should go.

I assumed it was obvious. I guess it isn't :-) Fixing.

Reply:
Actually, wait. We aren't exposing any knobs to control `auto-assign`.

I've added info about selecting which pool to use following this: https://github.com/kinvolk/lokomotive/pull/996#discussion_r492876707

I guess it doesn't make sense to mention disabling auto assignment as long as we don't expose this functionality.

Reply:
I've removed the section about `auto-assign`. We can discuss adding support for it in a new issue if there is demand for it (but there is a bunch of knobs we don't support at the moment).